### PR TITLE
add home page and tests

### LINKED ISSUE #1: Landing/home page
There should be a landing page for the application, with a prominent search bar. [Additional features for home page to come]

Acceptance criteria:
* I see a landing page and can start a search.
* The landing page does not look like the default Blacklight home page.

### PR DESCRIPTION
**Pull request by gordonleacock**
closes #1 
screen shot 2017-04-14 at 2 39 17 pm

### CODE REVIEW

File: app/views/catalog/_home.html.erb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<div class='jumbotron'>
+  <div class='row'>
+    <h1><%= t('arclight.views.home.heading') %></h1>
```
Comment (jkeck) [inline app/views/catalog/_home.html.erb]: What would you think about adding `text-center` class here to center this heading?

**Comment by jkeck** [comment]
Looks good. 👌 

One minor comment that you may have already thought about.

File: app/views/catalog/_home.html.erb [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+  </div>
+  <div class='col-md-6 col-md-offset-3'>
```
Comment (jkeck) [inline app/views/catalog/_home.html.erb]: I wonder if we should make this a bit larger on desktop (just using the bootstrap classes).  Perhaps that got more complicated than I'm making it sound, so totally understand if we want to punt and work on the proper sizing later.

Comment (gordonleacock) [inline app/views/catalog/_home.html.erb]: I'll see if I can add those modifications next.

**Pull request merged**